bool query
---
float32[] angles
float32[] intensities
float32[] diffs
float32[] anglesMajor
float32[] anglesMinor
